examples/measure.html

```diff
@@ -1,10 +1,12 @@
 ---
 template: example.html
 title: Measure example
-shortdesc: Example of using the ol.interaction.Draw interaction for creating simple measuring application.
+shortdesc: Example of using the ol.interaction.Draw interaction for creating a simple measuring application.
 docs: >
-  <p><i>NOTE: If use geodesic measures is not checked, measure is done in simple way on projected plane. Earth curvature is not taken into account</i></p>
-tags: "draw, edit, measure, vector"
+  <p><i>NOTE: Area and length calculations are done using ol.proj.getLength and ol.proj.getArea to get answers in meters or square meters at any latitude.
+  The ol.geom.Linestring.getLength method does not give a length in meters for EPSG:3857 or EPSG:4326 linestrings.
```

Comment:
This is confusing. Meters should always be used. Same with square meters below. These methods should always return meters or square meters.